src/Export/Adapters/PriceAdapter.php

```diff
@@ -19,12 +27,32 @@ class PriceAdapter
     /** @var ExportContext */
     protected $exportContext;
 
+    /** @var ProductPriceCalculator|null */
+    protected $calculator = null;
```

Comment:
This should never be `null`, as it should always exist

```suggestion
    protected $calculator;
```